codegens/powershell-restmethod/lib/index.js

```diff
@@ -293,8 +293,8 @@ function convert (request, options, callback) {
                         `${request.method}' -Headers $headers`;
   }
   else {
-    codeSnippet += `$response = Invoke-RestMethod '${request.url.toString()}' -CustomMethod ` +
-                        `'${request.method}' -Headers $headers`;
+    codeSnippet += `$response = Invoke-RestMethod '${request.url.toString().replace(/'/g, '\'\'')}' -CustomMethod ` +
```

Comment:
Let's separate out this .replace logic to a common function and utilise it instead of direct logic.